docroot/profiles/sdss/sdss_profile/themes/sdss_subtheme/sdss_subtheme.theme

```diff
@@ -123,5 +125,28 @@ function sdss_subtheme_preprocess_page(&$vars) {
   // These need to allows be the "dark" variants for the SDSS subtheme.
   $vars['brand_bar_variant'] = 'su-brand-bar--dark';
   $vars['global_footer_variant'] = 'su-global-footer--dark';
+
+  // Implement full-width basic page styles
+  if (
+    isset($vars['node']) &&
+    $vars['node'] instanceof NodeInterface &&
+    $vars['node']->bundle() == 'stanford_page' &&
+    $vars['node']->hasField('layout_selection') &&
+    $vars['node']->get('layout_selection')->getString() == 'stanford_basic_page_full'
```

Comment:
Are all these conditions necessary? If they are, the Drupal way is to separate them into separate variable conditions and use fail conditions with returns. For example:

```
$is_page_node = isset($vars['node']) && $vars['node'] instanceof NodeInterface && $vars['node']->bundle() == 'stanford_page';

if (!$is_page_node) {
  return;
}

$has_layout_field = $vars['node']->hasField('layout_selection') && $vars['node']->get('layout_selection')->getString() == 'stanford_basic_page_full';

if ($has_layout_field) {
  // do stuff here
}
```

See: https://www.drupal.org/docs/develop/standards/php/php-coding-standards#linelength

Reply:
> "Conditions should not be wrapped into multiple lines". I partly disagree, but hey, they established it so we should follow along

Yea, I also agree with you here especially if the conditions are straightforward. I do think the documentation mentions the complexity matters for that specific part. Stacking 5 conditions is probably the limit 😄 .